Modules/Quotas/src/Quotas.Infrastructure.Database.SqlServer/Migrations/20230419133243_CreatedIdentitiesTiersQuota.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Quotas.Infrastructure.Database.SqlServer.Migrations
+{
+    /// <inheritdoc />
+    public partial class CreatedIdentitiesTiersQuota : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
```

Comment:
Like the Postgres migration, this one is also empty.
In addition to that, both migrations should have similar names.